### ISSUE TITLE
Introduce committing flow

### ISSUE THREAD

**Issue by ennru** [akka/alpakka-kafka#619]
A non-API-breaking way to hide the commitScaladsl/commitJavadsl methods is to introduce a flow eg. Committer that does it behind the scenes.
It could offer pre-packaged groupedWithin and other useful patterns.

**Comment by ennru** [comment]
@rtimush Did you look into contributing your solution for this already?

**Comment by rtimush** [comment]
@ennru Yes, I started working on it, but then had a couple of busy weeks and didn't have time to finish. I will try to push something this week.
